opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -282,8 +283,10 @@ namespace detail {
                 current_relaxation_ -= nonlinear_solver.relaxIncrement();
                 current_relaxation_ = std::max(current_relaxation_, nonlinear_solver.relaxMax());
                 if (terminalOutputEnabled()) {
-                    std::cout << " Oscillating behavior detected: Relaxation set to "
-                              << current_relaxation_ << std::endl;
+                    std::string msg = " Oscillating behavior detected: Relaxation set to "
+                        + std::to_string(current_relaxation_);
+                    std::cout << msg << std::endl;
```

Comment:
This should not be necessary, the info() call below should be sufficient to print this to screen if that has been requested (and by default it should).